docs/src/main/sphinx/installation/cli.rst

```diff
@@ -143,6 +143,8 @@ mode:
     - Enables display of debug information during CLI usage for
       :ref:`cli-troubleshooting`. Displays more information about query
       processing statistics.
+  * - ``--disable-autosuggestion``
```

Comment:
Please change to `--disable-auto-suggestion`. The original parameter was wrong.

Reply:
please fix asap @Colebow so we can get the PR merged